src/components/Login/Login.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import { BaseColor } from "../../config/color";
+import { Type } from "../../shared/contexts/actions";
+import { useGeneralContext } from "../../shared/contexts/StoreProvider";
+import { useFindUser } from "../../shared/graphql/request/userRequest";
+import Button from "../Button/Button";
+
+const LoginComponent = () => {
+  const [email, setEmail] = useState("");
+  const [error, setError] = useState("");
+  const { findUser, errorRequest, data, loading } = useFindUser();
```

Comment:
question, why did you created a new hook for this request if you already have a hook from gql?

Reply:
because this increases more lines of code, and to make it more readable, also exist the possibility of reuse if you add the functionality of looking up a user